sources/platform/actors/running/actor_standby.md

```diff
@@ -1,16 +1,16 @@
 ---
-title: Actor Standby
-description: Use the Actor as a real-time API server.
+title: Standby mode
+description: Use an Actor as a real-time API server.
 sidebar_position: 7.3
 slug: /actors/running/standby
 sidebar_label: Actor Standby
 ---
 
-**Use Actors in lightweight mode as a blazingly fast API server.**
+**Use Actors in lightweight Standby mode for fast API respoonses.**
```

Comment:
there is a typo here: `respoonses`, cc @TC-MO

Reply:
ouch, my bad

Reply:
Curious it wasn't caught by Typos check, I think it might need some further tinkering. Thank you will fix it ASAP